### DISCUSSION TITLE
请教一下如何支持batch输入?

### DISCUSSION THREAD

**Discussion by whlook** [IDEA-Research/Grounded-Segment-Anything#188] [General]
如题，也可以在这里讨论哈

**Comment by whlook** [comment]
![image](https://user-images.githubusercontent.com/13912755/233885837-c3ff2adb-346e-4a18-af9f-e19818df2d6c.png)
尝试改动


发生错误。。。。
![image](https://user-images.githubusercontent.com/13912755/233885944-9cd0808d-3576-4d64-a412-46d1f3a8518a.png)

**Comment by rentainhe** [comment]
不好意思我们暂时还没有支持batch输入，这边需要支持batch输入的话可能需要先从GroundingDINO开始支持，这边我们会先着手完善GroundingDINO的Demo

**Reply by yuxyang88** [reply]
请问GroundingDINO 现在能支持 batch输入了吗？ 改的思路是把数据resize 或 padding成同样的size。这样 输入一个batch， model可以自动推理吗？

**Comment by yinian-lw** [comment]
@whlook 请问这个问题你解决了吗？求帮助！！！
